include/hpp/fcl/BV/AABB.h

```diff
@@ -234,9 +240,9 @@ static inline AABB rotate(const AABB& aabb, const Matrix3f& R)
 {
   AABB res (R * aabb.min_);
   Vec3f corner (aabb.min_);
-  const std::size_t bit[3] = { 1, 2, 4 };
-  for (std::size_t ic = 1; ic < 8; ++ic) { // ic = 0 corresponds to aabb.min_. Skip it.
-    for (std::size_t i = 0; i < 3; ++i) {
+  const Eigen::DenseIndex bit[3] = { 1, 2, 4 };
+  for (Eigen::DenseIndex ic = 1; ic < 8; ++ic) { // ic = 0 corresponds to aabb.min_. Skip it.
+    for (Eigen::DenseIndex i = 0; i < 3; ++i) {
       corner[i] = (ic && bit[i]) ? aabb.max_[i] : aabb.min_[i];
```

Comment:
Isn't this bugged ? I would expect this to be
```suggestion
      corner[i] = (ic & bit[i]) ? aabb.max_[i] : aabb.min_[i];
```

Reply:
I will fix it, even if it is not the scope of this PR initially.

Reply:
Thanks.